designs/administrator/future-openshift/channel-ux-improvements/index.md

```diff
@@ -0,0 +1,61 @@
+---
+parent: Administrator
+version: 4.8
+---
+
+# Improvements to channel states and statuses
+
+This unit of work is a part of our continuous effort to improve the upgrade experience. In particular, this set of enhancements focuses on improving how we communicate the state a channel is in, as well as the actions users can (or cannot) take to configure channels and receive update recommendations. Channel descriptions will be available in the channel dropdown to provide as much in context information as possible for users to make an informative decision on what channel to select based on their needs.
+
+The enhancements focus on improving the following scenarios:
+1. Create a more informative empty state for when a channel is not configured.
+2. Help users who end up without any available channel choices understand why.
+3. Help users recover from unknown versions.
+
+## Scenario 01 | Create a more informative empty state for when a channel is not configured
+
+![Current – No Channel Selected](img/Original-NoChannelSelected.png)
+- The current implementation (pictured above) for when a channel is not configured falls short in informing users the value of configuring a channel.
+
+![Proposed - No Channel Selected](img/New-NoChannelSelected-1.png)
+- The new design (picutred above) improves how we handle 'I'm not in any channels': 
+    1. The new message under **Update status** lets users know the impact of not configuring a channel and directly states under the **Channel** selector that one is "Not configured."
+    2. An informational alert was added to inform users that they are not requesting update recommendations and provides the action necessary for users to resolve. 
+
+- In addition to the improvements on the **Cluster setting details page** the **Input channel model** provides an input field for the channel. To verify if the current cluster version exists in the channel a user inputs, they must save and check the update status. 
+
+![Proposed - No Channel Selected Modal](img/inputchannel.png)
```

Comment:
Just curious and sorry if I am reading this incorrectly. Does the user have to type in one of the channels from the list (stable-4.8, fast...)? If that is the case I wonder if a dropdown might remove any chance of human error.